src/Symfony/Component/Console/Application.php

```diff
@@ -489,9 +489,6 @@ public function add(Command $command)
             return null;
         }
 
-        // Will throw if the command is not correctly initialized.
-        $command->getDefinition();
```

Comment:
This break laziness (#39851 should also fix this issue).
I'm not sure why this check is performed here.

Reply:
Related to https://github.com/symfony/symfony/pull/9244 apparently
Apparently, we need to think twice before removing as that might hurt DX.